lib/ontologies_linked_data/config/config.rb

```diff
@@ -49,7 +49,7 @@ def config(&block)
     @settings.fundedBy                      ||= {}
     @settings.federated_portals             ||= {}
 
-    @settings.ui_host                       ||= 'bioportal.bioontology.org'
+    @settings.ui_host                       ||= 'https://bioportal.bioontology.org'
```

Comment:
the host can not have `https` I think , why this change anyway ?

Reply:
because i'm putting the default value for :homepage to be `RDF::URI(LinkedData.settings.ui_host)` and this will make an erro in the test with agraph and graphdb

Reply:
this error:
```
SPARQL::Client::MalformedQuery: MALFORMED QUERY: No base URI for URI <bioportal.bioontology.org> Processing query INSERT DATA { GRAPH <https://w3id.org/mod#Semanticartefactcatalog> {
```

Reply:
i'll change it to put the scheme in the default value not the config one